### PR TITLE
feat(EMS-1273): Application submission - CSV generation enhancements/conditions

### PR DESCRIPTION
**Pull request by ttbarnes**
This PR updates the application CSV generation to conditionally render some rows/fields depending on the submitted answers

## Changes

- Conditionally map policy and export fields depending on if it's a single or multiple policy.
- Conditionally map exporter broker fields depending on if a broker is being used or not.
- Add helper functions required for formatting the answers.
- Add titles to each section generated in the CSV.

## Other improvements
- Tidy up constants imports/exports.
- Simplify some helper functions.

### CODE REVIEW

File: src/api/generate-csv/map-application-to-csv/helpers/map-month-string/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import mapMonthString from '.';
+
+describe('api/generate-csv/map-application-to-csv/helpers/map-month-string', () => {
+  describe('when value is 0', () => {
+    it('should return a formatted string with `month`', () => {
+      const result = mapMonthString(0);
```
Comment (abhi-markan) [inline src/api/generate-csv/map-application-to-csv/helpers/map-month-string/index.test.ts:6]: Assertions covering:
* `` (Empty)
* ABC
* !"£

File: src/api/generate-csv/map-application-to-csv/helpers/map-month-string/index.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/**
+ * mapMonthString
+ * Map credit period answer to a month/s string
+ * @param {Number} Credit period
+ * @returns {String} Credit period with month/s string
+ */
+const mapMonthString = (answer: number) => {
+  if (answer === 1) {
```
Comment (abhi-markan) [inline src/api/generate-csv/map-application-to-csv/helpers/map-month-string/index.ts:8]: Ternary operator will reduce number of line.

File: src/api/generate-csv/map-application-to-csv/helpers/policy-type/index.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { FIELD_VALUES } from '../../../../constants';
+
+/**
+ * isSinglePolicyType
+ * Check if a policy type is 'Single contract policy'
+ * @param {String} Policy type
+ * @returns {Boolean}
+ */
+const isSinglePolicyType = (policyType: string) => {
+  if (policyType === FIELD_VALUES.POLICY_TYPE.SINGLE) {
```
Comment (abhi-markan) [inline src/api/generate-csv/map-application-to-csv/helpers/policy-type/index.ts:10]: Ternary operator for reduced LOC.

File: src/api/generate-csv/map-application-to-csv/helpers/policy-type/index.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+/**
+ * isMultiPolicyType
+ * Check if a policy type is 'Multiple contract policy'
+ * @param {String} Policy type
+ * @returns {Boolean}
+ */
+const isMultiPolicyType = (policyType: string) => {
+  if (policyType === FIELD_VALUES.POLICY_TYPE.MULTIPLE) {
+    return true;
```
Comment (abhi-markan) [inline src/api/generate-csv/map-application-to-csv/helpers/policy-type/index.ts:25]: Ternary please.

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Looks good, few changes to address please.

File: src/api/generate-csv/map-application-to-csv/helpers/policy-type/index.ts (hunk 3 above)
Comment (ttbarnes) [reply src/api/generate-csv/map-application-to-csv/helpers/policy-type/index.ts:10]: Ternary actually not required, made it even simpler 😎

File: src/api/generate-csv/map-application-to-csv/helpers/map-month-string/index.ts (hunk 2 above)
Comment (ttbarnes) [reply src/api/generate-csv/map-application-to-csv/helpers/map-month-string/index.ts:8]: I think a one liner makes it harder to read but have made the change.

File: src/api/generate-csv/map-application-to-csv/helpers/map-month-string/index.test.ts (hunk 1 above)
Comment (ttbarnes) [reply src/api/generate-csv/map-application-to-csv/helpers/map-month-string/index.test.ts:6]: We don't need to do this, typescript prevents anything other than a number from being passed to the function. If e.g a string is passed the code will not compile.

**Review by abhi-markan — APPROVED** [review]

**Review by Zainzzkk — APPROVED** [review]

**Pull request merged**